editor/src/styles/_login.scss

```diff
@@ -44,9 +62,36 @@ berta-login {
       &.google {
```

Comment:
Please DRY your css. Add only specific css properties in `google` and `facebook` classes. All other common properties should be shared from `social-login a`.